test/test_cli.rb

```diff
@@ -1,441 +1,357 @@
 # frozen_string_literal: true
+
 require_relative 'helper'
 require 'sidekiq/cli'
+require 'sidekiq/launcher'
```

Comment:
Maybe it's worth to require launcher at the top of `cli.rb`, but right now you get fail due to missing constant if you run `ruby test/test_cli.rb` solely.

Reply:
I believe cli should require launcher as it uses that class directly

Reply:
I'm definitely going to cover `#run` with tests ^_^ checkout the code precisely at the moment and think how to do it better. I guess it's possible to test some parts for non-daemonized mode with corresponding refactoring and mocking. Do you have the suggestions?